### PR TITLE
edit README and CONTRIBUTING

### PR DESCRIPTION
**Pull request by willingc**
- Edit key documents now that the monorepo is in place.
- clean up markdown formatting

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -25,7 +26,9 @@ advised to understand the overall system.
 * [ZeroMQ](http://zguide.zeromq.org/page:all)
 
 If you want a gentle guide to Rx + Jupyter messaging at the same time, we have
-a [build your own REPL with enchannel](https://github.com/nteract/docs/blob/master/enchannel/build-your-own-repl.md) tutorial. This allows you to work without React while learning concepts, leading to implementing a light version of [ick](https://github.com/nteract/ick),
+a [build your own REPL with enchannel](https://github.com/nteract/docs/blob/master/enchannel/build-your-own-repl.md)
+tutorial. This allows you to work without React while learning concepts,
+leading to implementing a light version of [ick](https://github.com/nteract/ick),
```
Comment (rgbkrk) [inline CONTRIBUTING.md:31]: Oh man, these are so outdated now but way easier to implement.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -61,15 +78,13 @@ Good commit messages serve at least three important purposes:
...
 * To help us write a good release note.
 
-* To help the future maintainers of nteract/nteract (it could be you!),
-say five years into the future, to find out why a particular change was made to
-the code or why a specific feature was added.
+* To help the future maintainers of nteract/nteract (it could be you!), say
+  five years into the future, to find out why a particular change was made to
+  the code or why a specific feature was added.
 
 Structure your commit message like this:
```
Comment (rgbkrk) [inline CONTRIBUTING.md:85]: All awesome here.

File: CONTRIBUTING.md (hunk 1 above)
Comment (willingc) [reply CONTRIBUTING.md:31]: Now that enchannel is in the monorepo. Any additional changes here?

Comment (rgbkrk) [reply CONTRIBUTING.md:31]: We definitely need to spend some time writing that build-your-own-repl tutorial over with the new API, so I guess we can flag that as an issue to tackle. I think we can leave this paragraph as you've worded it for now.

Comment (willingc) [reply CONTRIBUTING.md:31]: Makes sense to me.

File: README.md [hunk 3]
```diff
@@ -7,43 +7,70 @@
...
+applications/ -- all the user facing applications (i.e. desktop, play)
+```
 
-### Development
+The `packages` directory has the components needed to build new applications,
+and the `applications` has the desktop app, the play app, and a few more.
+
+*Why have a monorepo?* The monorepo contains many components and packages that
+can be mixed and remixed to create new applications. The monorepo keeps these
+elements together so they are easy to discover and use.
```
Comment (rgbkrk) [inline README.md]: I'd add on that it makes it easy to iterate on applications that share common components. As an example, when we update the our jupyter message handling, if we were changing it for the desktop app and happened to break it for use by play.nteract.io we would notice them in tandem.

File: README.md [hunk 4]
```diff
@@ -52,29 +79,34 @@ To get started developing, [set up the nteract monorepo](#set-the-monorepo-up-in
...
 4. `npm install`
 
-#### Building specific packages
+#### Building a specific package
 
-In some cases you'll want to modify a base package (and not rebuild everything). To target just one use this, replacing `packageName` with the package you want to hack on.
+In some cases you'll want to modify an individual base package (i.e. commutable
+or transforms) and not rebuild all of the other packages. To target a build of a
+specific package, use this command, replacing `packageName` with the package you
+want to hack on:
```
Comment (rgbkrk) [inline README.md:96]: 👍

**Review by rgbkrk — APPROVED** [review]

**Pull request merged**